index.html

```diff
@@ -9222,6 +9222,240 @@ <h3 id="example-2">
+          </p>
+          <p>
+            Computing the makeup gain means executing the following steps:
+          </p>
+          <ol>
+            <li>Let <var>full range gain</var> be the value returned by putting
+            1.0 through the <a>compression curve</a>.
+            </li>
+            <li>Let <var>full range makeup gain</var> the inverse of <var>full
+            range gain</var>.
+            </li>
+            <li>Return the result of computing <var>full range makeup
+            gain</var><sup>0.6</sup>.
+            </li>
+          </ol>
+          <p>
+            Computing the <dfn>enveloppe rate</dfn> is done by putting the
+            ratio of the <var>compressor gain</var> and the <var>detector
+            average</var> through a curve. User-agents are allowed to choose
+            the shape of the attack and release curve. However, they MUST
+            respect the following constraints:
+          </p>
+          <ul>
+            <li>The input to the curve MUST be the ratio of the <var>compressor
+            gain</var> and the <var>detector average</var>.
+              <div class="note">
+                When attacking, this number inferior or equal to 1, when
+                releasing, this number is strictly superior to 1.
+              </div>
+            </li>
+            <li>The attack curve MUST be a continuous, monotonically increasing
+            function in the range \([0, 1]\).
+            </li>
+            <li>The release curve MUST be a continuous, monotonically
+            decreasing function that is always superior to 1.
+            </li>
+          </ul>
+          <div class="note">
+            It is allowed, for example, to have a compressor that performs an
+            <em>adaptative release</em>, that is, releasing faster the harder
+            the compression, or to have curves for attack and release that are
+            not of the same shape.
+          </div>
+          <p>
+            Putting a the absolute value of a sample through a <dfn>compression
+            curve</dfn> means computing the absolute value of this sample to a
+            function, and returning the computed value. This function MUST
+            respects the following characteristics:
+          </p>
+          <ol>
+            <li>This function is the identify up to the value of the linear
+            <code>threshold</code> (i.e., \(f(sample) = sample\)).
+            </li>
+            <li>From the <code>threshold</code> up to the <code>threshold +
+            knee</code>, User-Agents can choose the curve shape. The whole
+            function MUST be continuous (except as noted in the note below) and
+            monotonically increasing.
+              <div class="note">
+                As such, if the <code>knee</code> is 0, the
+                <a>DynamicsCompressorNode</a> is a hard-knee compressor, and
+                the function has a discontinuity at <code>threshold</code>.
```

Comment:
I think the MUST applies to the whole sentence: it MUST (be continuously differentiable except when the knee is 0, and then that is the case there is a discontinuity at the knee).

I've changed the prose a bit.